lib/oxidized/model/firebrick.rb

```diff
@@ -0,0 +1,32 @@
+class Firebrick < Oxidized::Model
+  # Firebrick #
+  prompt /\x0a\x1b\x5b\x32\x4b\x0d.*>\s/
+
+  cmd :all do |cfg|
+    # sometimes ironware inserts arbitrary whitespace after commands are
```

Comment:
Does Firebrick have any relation to ironware? Is this copypasta from `ironware.rb`?